File: tests/fixtures/javascript/input.js
// Line comment.
/**
 * JSDoc block.
 */
function main() {
  /* inline block */
  const url = "https://example.com"; // trailing
  const s = 'not // a comment';
  const t = `template /* not a comment */`;
  return [url, s, t];
}

File: tests/fixtures/rust/input.rs
//! Module doc comment.
// A line comment.

/// Doc comment on main.
fn main() {
    /* nested /* block */ comment */
    let url = "https://example.com/#anchor"; // trailing comment
    let s = "not // a comment";
    /* multi
       line
       block */
    println!("{url} {s}");
}
